tests/models/whisper/test_modeling_whisper.py

```diff
@@ -853,9 +853,9 @@ def test_large_logits_librispeech(self):
         input_speech = self._load_datasamples(1)
 
         processor = WhisperProcessor.from_pretrained("openai/whisper-large")
-        processed_inputs = processor(audio=input_speech, text="This part of the speech", return_tensors="pt")
+        processed_inputs = processor(audio=input_speech, return_tensors="pt")
```

Comment:
Would personally prefer that we don't hardcode the labels and instead tokenise the target text. We can set `add_special_tokens` to False if we don't want the 'special' BOS and EOS tokens, as we do here:
https://github.com/huggingface/transformers/blob/cbbeca3d1733aa7c9b443af5ff231a5affcd8a1e/tests/models/whisper/test_tokenization_whisper.py#L120

```suggestion
        processed_inputs = processor(audio=input_speech, text="This part of the speech", add_special_tokens=False, return_tensors="pt")
```